oca/pool.py

```diff
@@ -160,3 +160,11 @@ def delete(self):
         '''
         self.client.call(self.METHODS['delete'], self.id)
 
+    def clone(self, name=''):
+        '''
+        Creates a clone of an elemet
+        ``name``
+            name of a target element
+        '''
+        self.client.call(self.METHODS['clone'], self.id, name)
```

Comment:
Actually I took a look in OpenNebula documentation.
The `one.image.clone` method has an additional param:
```
The ID of the target datastore. Optional, can be set to -1 to use the current one.
```
So I am not sure if it's ok to have a clone method inside a base class.
Nevertheless you can have it in base class, just add a separate implementation for the image clone.